db/seeds.rb

```diff
@@ -1,5 +1,32 @@
 locations = {
   meetups: [
+    {
+      slug: 'ruby-tuesday-tallinn',
+      name: 'Ruby Tuesday Tallinn',
+      description: 'Ruby User Group meeting every last Tuesday of the month from 18:00 onward.',
+      latitude: 59.398279,
+      longitude: 24.659693,
+      link: 'https://www.facebook.com/events/785659903522120',
+      img: './ruby-tuesday-tallinn.jpg'
+    },
+    {
+      slug: 'ruby-belgium',
+      name: 'Ruby Belgium',
+      description: 'Ruby Belgium is a community of programming enthusiasts using ruby as a tool to connect, build and share knowledge',
+      latitude: 50.84792799176271,
+      longitude: 4.3534307855468235,
+      link: 'https://rubybelgium.be/',
+      img: 'https://rubybelgium.be/assets/images/ruby-ecc3a010.svg'
```

Comment:
could you please download this logo as well? We are trying to have all the logos inside of the repository to not rely on the remote servers. Same story - put in public/logos and link with `/logos/ruby-ecc3a010.svg `

Thank you!